config/prod/kristen-bucket.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "kristen-bucket"
+sceptre_user_data:
+  # (Optional) Synapse IDs for owner.txt file
+  SynapseIDs: ["1968150"]  # Kristen Dang
```

Comment:
@kkdang This synapse owner id will get set in the bucket owner.txt file which means that you are the only one that has access to this bucket from synapse.  Please let us know if you want to add/remove IDs from here.

Reply:
Xengie Doan will be executing the data transfer, so she will also need access.

Reply:
ohh well, @xdoan is familiar with this process and knows how to correctly provision buckets with our system.  It would be easier if we could have her do the PR instead because she will know the parameter values to provide.